container/test-app/compound/compoundClientAPI.html

```diff
@@ -204,6 +211,31 @@ <h3>
       compoundContainer.addEventListener(MFEventID.SET_VIEW_GROUP_DATA_REQUEST, event => {
         console.log('Set View Group Data Request received', event.detail, event);
       });
+
+      const deferInitContainer = document.getElementById('defer-init-test');
+      const button = document.getElementById("init-button");
+      button.addEventListener("click", function() {
+        deferInitContainer.init();
+      });
+      deferInitContainer.compoundConfig = {
+        renderer: {
+          use: 'grid',
+          config: {
+            columns: '1fr 1fr',
+            gap: '20px'
+          }
+        },
+        children: [
+          {
+            viewUrl: './compound/myCompoundWebComponent1.js'
+          },
+          {
+            viewUrl: './compound/myCompoundWebComponent2.js'
```

Comment:
1. **myCompoundWebComponent1** is too generic. To avoid clutter please rename the file to **compoundWCDeferInit1** and **compoundWCDeferInit2** or similar.

2. Also since it seems like we will add new test wc.js files in the future put them in a folder as well to avoid confusion:  (example:)
```suggestion
            viewUrl: './compound/defer-init-wc/compoundWCDeferInit1.js'
          },
          {
            viewUrl: './compound/defer-init-wc/compoundWCDeferInit2.js'
```